lib/hyrax/valkyrie_can_can_adapter.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Hyrax
+  ##
+  # A `CanCan::ModelAdapter` for valkyrie resources
+  class ValkyrieCanCanAdapter < CanCan::ModelAdapters::AbstractAdapter
```

Comment:
Does it make sense to have specific tests for this class or just let it be tested through ability tests?

Reply:
unit tests are probably good. i'll look into what's needed here.

Reply:
i pushed up unit tests for this.